src/Honeybee_infORventPerArea Calculator.py

```diff
@@ -0,0 +1,175 @@
+# ACH 2 m3/s-m2
+#
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
+# 
+# This file is part of Honeybee.
+# 
+# Copyright (c) 2013-2016, Abraham Yezioro <ayez@ar.technion.ac.il> 
+# Honeybee is free software; you can redistribute it and/or modify 
+# it under the terms of the GNU General Public License as published 
+# by the Free Software Foundation; either version 3 of the License, 
+# or (at your option) any later version. 
+# 
+# Honeybee is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+
+"""
+Use this component to transform ACH to m3/s-m2.
+Plug the result to the Honeybee setEPZoneLoads component, infiltrationRatePerArea_ or  infiltrationRatePerArea_ inputs
+
+-
+Provided by Honeybee 0.0.59
+    
+    Args:
+        _HBZones: Honeybee zones for which you want to calculate the infiltration or ventilation rates.
+        _airChangeHour: Air Changes per Hour. Given at ambien pressure. Give a value of 0.2 or higher.
+        _at50PA_: Boolean. Set to true if you want to get a value at 50 Pascal. You probably want this for PassiveHouse calculations. Default is False as for ambient calculation.
+    Returns:
+        readMe!: Report of the calculations
+        infORventPerArea: infiltrationRatePerArea or ventilationPerArea in m3/s-m2 (Cubic meters per second per square meter of floor)
+"""
+ghenv.Component.Name = "Honeybee_infORventPerArea Calculator"
```

Comment:
Make sense. Then we need to remove the older component. I think I can take care of that. Thanks!